packages/trip-form/src/TripOptions/FeaturedOption.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  accessModeIsWalkOnly,
+  getCategoryModes,
+  getCategoryPrimaryMode,
+  getNonTransitModes,
+  getSelectedModes
+} from "./util";
+import * as S from "./styled";
+
+const FeaturedOption = ({
+  onQueryParamChange,
+  queryParams,
+  questionIcon,
+  setFeaturedOption,
+  supportedModes,
+  DetailedModeIcon
+}: QueryProps & {
+  setFeaturedOption(option: string): void;
+  questionIcon?: React.ReactElement;
+}): React.ReactElement => {
+  // No featured mode to show if walking to transit
+  if (accessModeIsWalkOnly(queryParams?.mode)) return null;
+  const nonTransitModes = getNonTransitModes(queryParams?.mode);
+  const selectedCompanies = queryParams?.company?.split(",") || [];
+  const selectedModes = getSelectedModes(queryParams);
+  const option = nonTransitModes[0];
+  const category = supportedModes.categories.find(c =>
+    getCategoryModes(c).some(o => o === option)
+  );
+  const optionsAreCheckboxes = Boolean(category.mode);
+  return (
+    <S.FeaturedOptionContainer>
+      <div>
+        {category.options.map((o, index) => {
+          const companyIsSelected = selectedCompanies.some(
+            c => c === o.company
+          );
+          const modeIsSelected = selectedModes.some(m => m === o.mode);
+          const isChecked = optionsAreCheckboxes
+            ? companyIsSelected
+            : o.company
+            ? companyIsSelected && modeIsSelected
+            : modeIsSelected;
+          const selectOption = () => {
+            let mode = selectedModes;
+            let company = selectedCompanies;
+            if (isChecked) {
+              // Un-check the company box if dealing with checkboxes. Otherwise, do nothing.
+              if (optionsAreCheckboxes)
+                company = selectedCompanies.filter(c => c !== o.company);
+              // Do nothing if already radio button is already checked.
+              else return;
+            } else {
+              // if un checked, set/add company and set mode (FIXME: what about car/walk)
+              if (o.mode) {
+                mode = selectedModes
+                  .filter(coreUtils.itinerary.isTransit)
+                  .concat([o.mode]);
+              }
+              if (o.company) {
+                company = optionsAreCheckboxes
+                  ? selectedCompanies.concat([o.company])
+                  : [o.company];
+              }
+            }
+            onQueryParamChange({
+              company: company.join(","),
+              mode: mode.join(",")
+            });
+          };
+          return (
+            <OptionButton
+              checked={isChecked}
+              key={index}
+              label={o.label}
+              onClick={selectOption}
+              selected={isChecked}
+            />
+          );
+        })}
+      </div>
+      <div>
+        <S.QuestionButton onClick={() => setFeaturedOption(option)}>
+          {questionIcon || <QuestionCircle />}
+        </S.QuestionButton>
```

Comment:
Apologies if this has already been conveyed, but one of the asks from the last demo was to have the button that launches the overlay encapsulate both the question icon and the mode icon/image so clicking anywhere in that area would launch the overlay

Reply:
Don't worry at all about mentioning it again! Support added in 663f4b7. I've also had the question icon highlight when you hover over that zone, please let me know if this isn't expected behavior!

Reply:
Hey @miles-grant-ibigroup, sorry for the late response, we were wrapping and demoing our latest sprint for trimet.org towards the end of last week and then I was out Friday.  And yes that's right, the question icon should be highlighted when any part of the container holding it and the vehicle image are hovered over.

I believe we have access to our Jira and Confluence setup for you and @binh-dam-ibigroup so take a look at that if you get a chance, as we've started to document some of the details there